packages/components-native/src/Typography/Typography.style.ts

```diff
@@ -229,52 +232,267 @@ export const typographyTokens: { [index: string]: TextStyle } = {
+    color: tokens["color-informative--surface"],
+  },
+
+  informativeOnSurface: {
     color: tokens["color-informative--onSurface"],
   },
 
-  critical: {
+  // To be uncommented once success in Deprecated is removed
+  // success: {
+  //   color: tokens["color-success"],
+  // },
+
+  successSurface: {
+    color: tokens["color-success--surface"],
+  },
+
+  successOnSurface: {
+    color: tokens["color-success--onSurface"],
+  },
+
+  // Interactions
+
+  interactive: {
+    color: tokens["color-interactive"],
+  },
+
+  interactiveHover: {
+    color: tokens["color-interactive--hover"],
+  },
+
+  interactiveSubtle: {
+    color: tokens["color-interactive--subtle"],
+  },
+
+  interactiveSubtleHover: {
+    color: tokens["color-interactive--subtle--hover"],
+  },
+
+  destructive: {
+    color: tokens["color-destructive"],
+  },
+
+  destructiveHover: {
+    color: tokens["color-destructive--hover"],
+  },
+
+  disabled: {
+    color: tokens["color-disabled"],
+  },
+
+  disabledSecondary: {
+    color: tokens["color-disabled--secondary"],
+  },
+
+  surface: {
+    color: tokens["color-surface"],
+  },
+
+  // Workflow
+
+  request: {
+    color: tokens["color-request"],
+  },
+
+  requestSurface: {
+    color: tokens["color-request--surface"],
+  },
+
+  requestOnSurface: {
+    color: tokens["color-request--onSurface"],
+  },
+
+  quote: {
+    color: tokens["color-quote"],
+  },
+
+  quoteSurface: {
+    color: tokens["color-quote--surface"],
+  },
+
+  job: {
+    color: tokens["color-job"],
+  },
+
+  jobSurface: {
+    color: tokens["color-job--surface"],
+  },
+
+  jobOnSurface: {
+    color: tokens["color-job--onSurface"],
+  },
+
+  visit: {
+    color: tokens["color-visit"],
+  },
+
+  visitSurface: {
+    color: tokens["color-visit--surface"],
+  },
+
+  visitOnSurface: {
+    color: tokens["color-visit--onSurface"],
+  },
+
+  task: {
+    color: tokens["color-task"],
+  },
+
+  taskSurface: {
+    color: tokens["color-task--surface"],
+  },
+
+  taskOnSurface: {
+    color: tokens["color-task--onSurface"],
+  },
+
+  event: {
+    color: tokens["color-event"],
+  },
+
+  eventSurface: {
+    color: tokens["color-event--surface"],
+  },
+
+  eventOnSurface: {
+    color: tokens["color-event--onSurface"],
+  },
+
+  invoice: {
+    color: tokens["color-invoice"],
+  },
+
+  invoiceSurface: {
+    color: tokens["color-invoice--surface"],
+  },
+
+  invoiceOnSurface: {
+    color: tokens["color-invoice--onSurface"],
+  },
+
+  payments: {
+    color: tokens["color-payments"],
+  },
+
+  paymentsSurface: {
+    color: tokens["color-payments--surface"],
+  },
+
+  paymentsOnSurface: {
+    color: tokens["color-payments--onSurface"],
+  },
+
+  client: {
+    color: tokens["color-client"],
+  },
+
+  // Miscellaneous
+
+  icon: {
+    color: tokens["color-icon"],
+  },
+
+  brand: {
+    color: tokens["color-brand"],
+  },
+
+  // Deprecated
+
+  blue: {
+    color: tokens["color-heading"],
+  },
+
+  base: {
```

Comment:
I could see the argument that we don't want to deprecate `base` and `baseReverse` even though they are duplicates. I think we should strive to have no duplicates to cut down on confusion and make things clearer when comparing code but if folks think removing `base` is a step too far I can remove it from this section